include/utils.h

```diff
@@ -504,8 +494,8 @@ inline void prune_truthset_for_range(const std::string &bin_file, float range,
     reader.read((char *)&npts_i32, sizeof(int));
```

Comment:
It would also be nice to use the get_metadata (and similar functions) to avoid repeated code